provider/provider.go

```diff
@@ -79,7 +80,9 @@ func (p *BaseProvider) getConfiguration(defaultTemplateFile string, funcMap temp
 		return nil, err
 	}
 
-	if _, err := toml.Decode(buffer.String(), configuration); err != nil {
+	var renderedTemplate = buffer.String()
+	log.Debugf("Rendering results of %s:\n%s", defaultTemplateFile, renderedTemplate)
```

Comment:
Not sure if this is something that is desired, but it certainly helped me debug while coding this.